src/lib/functions/runtimes/js/worker.mjs

```diff
@@ -0,0 +1,52 @@
+import { createServer } from 'net'
+import { isMainThread, workerData, parentPort } from 'worker_threads'
+
+import lambdaLocal from '@skn0tt/lambda-local'
+import { isStream } from 'is-stream'
+import sourceMapSupport from 'source-map-support'
+
+import { SECONDS_TO_MILLISECONDS } from './constants.mjs'
+
+if (isMainThread) {
```

Comment:
When would this happen?

Reply:
Never on purpose, but maybe during development somebody would import it. And then this error makes it obvious that that's not how it's intended to be used. The Node.js docs also always use this: https://nodejs.org/api/worker_threads.html

But we can also remove it and add a big header comment instead, your call.

Reply:
Nah, that makes sense. Just wanted to double-check.